wgsl/index.bs

```diff
@@ -542,6 +542,18 @@ A <dfn>literal</dfn> is one of:
 * A <dfn>numeric literal</dfn>: either an [=integer literal=] or a [=floating point literal=],
     and is used to represent a number.
 
+<div class='syntax' noexport='true'>
+  <dfn for=syntax>const_literal</dfn> :
```

Comment:
As you suggested in the editors meeting.
```suggestion
  <dfn for=syntax>literal</dfn> :
```

Reply:
@alan-baker thank you very much, please check the new commit!